src/Test.FunctionalTests.Sarif/UpdateBaselines.ps1

```diff
@@ -1,112 +0,0 @@
-﻿Param(
-    [string]$ToolName
```

Comment:
>$ToolName [](start = 12, length = 9)

Great to see this file disappear. Do we have some test output that provides guidance on how to rebaseline? Ideally, this information would appear in test output. I assume that you've gone through the experience of having the converter tests fail and have verified that all the test output is complete and works? (i.e., that the generated file diff command isn't broken, as I assume it wouldn't be).

Reply:
Yes. There's a base class for the E2E tests. It writes a message, which I updated, which says to copy the folder (and gives the full path) to update the tests.